src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala

```diff
@@ -560,6 +561,27 @@ class AddDatasetMetadataToDepositSpec extends UnitSpec with BeforeAndAfterAll {
     verify(datasetToXml(dataset), expectedXml)
   }
 
+  it should "return the expected xml with streaming surrogate if it is an A/V dataset" in {
+    object XmlTransform extends RewriteRule {
+      override def transform(n: Node): Seq[Node] =
+        n match {
+          case Elem(prefix, "profile", attribs, scope, children @ _*) =>
+            Elem(prefix, "profile", attribs, scope, false, children ++
+              <ddm:relation scheme="STREAMING_SURROGATE_RELATION">
+                /domain/randomdomainname/user/randomusername/collection/randomcollectionname/presentation/$presentation-placeholder
+              </ddm:relation> :_*)
```

Comment:
What do you mean? The XML element? Doesn't really matter. I'm not testing for formatting. That's removed in the test below.